bin/common.bash

```diff
@@ -102,11 +118,7 @@ validate_version() {
 
 # Check if the cloud provider is supported.
 validate_cloud() {
-    if [ "${1}" != "exoscale" ] &&
-       [ "${1}" != "safespring" ] &&
-       [ "${1}" != "citycloud" ] &&
-       [ "${1}" != "baremetal" ] &&
-       [ "${1}" != "aws" ]; then
+    if ! array_contains "${1}" "${ck8s_cloud_providers[@]}"; then
         log_error "ERROR: Unsupported cloud provider: ${1}"
```

Comment:
Should we add the list of supported providers here?
```suggestion
        log_error "ERROR: Unsupported cloud provider: ${1}"
        log_error "Supported providers: ${ck8s_cloud_providers[@]}"
```

Reply:
What do you think about `log_error: "Use the command **ck8s providers** to list supported providers"? It's fine with me either way.

Reply:
Yea, that good as well :slightly_smiling_face:

Reply:
Nevermind, I think proposing commands here could become a problem because that kind of assumes that this is the top-level entrypoint. In the devbox `ck8s` is renamed to `ck8s-apps` and that message would only be confusing in that case. I'll just list the supported providers like you proposed! :slightly_smiling_face:

Reply:
Pushed!